source/assets/public/index.css

```diff
@@ -216,7 +219,7 @@ body {
   }
   .breakpoint-desktop-up .body-inner-wrapper h2 {
     font-size: 3rem;
-    letter-spacing: .4rem;
+    letter-spacing: .75rem;
```

Comment:
the .75rem was initially to make it look closer to the sketch layout but I'm happy to keep it at .4rem :)